packages/shared_preferences/example/integration_test/shared_preferences_test.dart

```diff
@@ -101,13 +101,13 @@ void main() {
 
     testWidgets('simultaneous writes', (WidgetTester _) async {
       final List<Future<bool>> writes = <Future<bool>>[];
-      final int writeCount = 100;
```

Comment:
Shouldn't it be similar to the original if possible? Is this fragmentation worth the risk?
(I'm just curious what other people think regrading this topic)

Reply:
I personally don't care because replacing `analysis_options.yaml` is going on in the flutter/plugins repo (https://github.com/flutter-tizen/plugins/issues/161) and this integration test file will be updated soon when the migration is done.